File: class.cpp
#include <iostream>
#include <string>
using namespace std;

class Car {
public:
    string brand;
    string model;
    int year;

    Car(string brand, string model, int year) {
        this->brand = brand;
        this->model = model;
        this->year = year;
    }

    void showDetails() {
        cout << "Brand: " << brand << endl;
        cout << "Model: " << model << endl;
        cout << "Year: " << year << endl;
    }
};

int main() {
    Car toyota("Toyota", "Corolla", 2010);
    toyota.showDetails();

    return 0;
}

File: class.py
# Decleare class
class Car:
    def __init__(self, brand, model, year):
        self.brand = brand
        self.model = model
        self.year = year

    def show_details(self):
        print(f'Brand : {self.brand}/nModel : {self.model}/Year : {self.year}')


# creating object for Car

toyota = Car('Toyota', 'Corola', 2010)
toyota.show_details()